### PR TITLE
Docs props table updates

### PR DESCRIPTION
**Pull request by dbanksdesign**
#### Description of changes

https://docs-props-table-updates.d2q023dzpor7ad.amplifyapp.com/ (still deploying)

* Removing the rewrite rule for react component pages and moving to URL params `?tab=props` for the props tab. This required a lot small changes to MDX files moving around and such.
* Small visual improvements to how the props table renders

| Before | After |
| - | - |
|CleanShot 2022-06-28 at 16 21 55 | CleanShot 2022-06-28 at 16 33 57 |
| CleanShot 2022-06-28 at 16 34 40@2x | CleanShot 2022-06-28 at 16 34 28@2x |
|  Simulator Screen Shot - iPhone 13 - 2022-06-28 at 16 39 24 | Simulator Screen Shot - iPhone 13 - 2022-06-28 at 16 40 02 |

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are updated
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by reesscot — DISMISSED** [review]
LGTM

File: docs/src/components/Layout/PageTabLayout.tsx [hunk 1]
```diff
@@ -5,30 +5,32 @@ import { useCustomRouter } from '@/components/useCustomRouter';
...
+    query: { tab = '', platform },
     pathname,
     push,
   } = useCustomRouter();
-  const tabIndex = tabComponents.findIndex(
-    ({ title }) => title.toLowerCase() === tab
-  );
-  const changeURL = (e) => {
+
+  const defaultIndex = tab === 'props' ? 1 : 0;
```
Comment (joebuono) [inline docs/src/components/Layout/PageTabLayout.tsx:13]: Nice!

Comment (joebuono) [inline docs/src/components/Layout/PageTabLayout.tsx:13]: Nice!

**Review by joebuono — APPROVED** [review]
LGTM! Thanks for doing this

**Review by reesscot — APPROVED** [review]

**Comment by 0618** [comment]
why do we want to use URL param instead of the path? I think we prefer to use path for SEO

**Review by 0618 — APPROVED** [review]

**Pull request merged**